src/benchq/problem_ingestion/molecule_instance_generation.py

```diff
@@ -14,58 +18,61 @@
 )
 from openfermionpyscf import PyscfMolecularData
 from openfermionpyscf._run_pyscf import compute_integrals
+from orquestra import sdk
 from pyscf import gto, mp, scf
 
+from ..mlflow import _flatten_dict, create_mlflow_scf_callback
 
-class SCFConvergenceError(Exception):
-    pass
 
+def truncate_with_avas(
+    mean_field_object: scf.hf.SCF,
+    ao_list: Optional[Iterable[str]] = None,
+    minao: Optional[str] = None,
+) -> Tuple[gto.Mole, scf.hf.SCF]:
+    """Truncates a meanfield object to a specific active space that captures the
+    essential chemistry.
 
-@dataclass
-class ChemistryApplicationInstance:
-    """Class for representing chemistry application instances.
+    Args:
+        mean_field_object: The meanfield object to be truncated.
+        ao_list: A list of atomic orbitals to use for AVAS.
+        minao: The minimum active orbital to use for AVAS.
+    """
+    mean_field_object.verbose = 4
```

Comment:
Oh looks like it hasn't been touched since you committed 2ece50544c474fefbbc1a9f8dcdf5f22885ced45 😆. @AthenaCaesura @pediejo you were also listed as co-authors on the commit, is it okay if I remove that `mean_field_object.verbose = 4`?